### ISSUE TITLE
Sort by addition date

### ISSUE THREAD

**Issue by MrSenci** [arnesongit/repository.arneson#5]
Dear Arne, would you please integrate a new track listing format instead of track title?
I need one to list the tracks by addition date, thanks!

**Comment by NielsMayer** [comment]
I agree that sorting by last added would be a very useful and necessary feature.

**Comment by neukrisz** [comment]
+1
